.gitignore

```diff
@@ -2,4 +2,6 @@
 *.DS_Store
 build/
 dist/
-opmd_viewer.egg-info/
\ No newline at end of file
+opmd_viewer.egg-info/
+.cache/
+.eggs/
```

Comment:
if you want, adding an empty newline in all text/code files makes `diff`s better readable